Solutions/DNS Domain Solution for Microsoft Sentinel/Playbooks/SummarizeData/readme.md

```diff
@@ -0,0 +1,27 @@
+# DNS Domain Solution for Microsoft Sentinel Summarization capability
+
+This logic app helps to summarize DNS data into custom tables. Please note - enabling this playbook would incur additional cost.
+
+ ## Summary
+ To ensure good performance of DNS Domain solution, summarization capability can be used. This would create various custom tables containing analytics based on different parameters of ASIM DNS Schema.
+
+### Deployment instructions 
+1. Deploy the playbook by clicking on "Deploy to Azure" button. This will take you to deploying an ARM Template wizard.
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/deploySummarizationPublic)
```

Comment:
@vakohl, please check if we need to create a short link for these, thanks.

Reply:
@v-sabiraj yes, we do need to create it. Please refer to the playbook readme file for more details

Reply:
@v-sabiraj Did we create this short link?